### ISSUE TITLE
Tab through TOC and home page posts

### ISSUE THREAD

**Issue by ZacWarham** [halogenica/beautifulhugo#812]
We are reviewing the accessibility of our website and noticed that when using Tab to navigate our home page it goes in this order

1. Menu
2. Posts
3. TOC

With pagination set to 5, this means that you go through the first 5 posts, then the whole table of contents list, which includes the initial 5 posts again.

I am not entirely sure if this is related to this template or Hugo itself, but it seems to me that the TOC should be first or ignored entirely. But then I am far from an expert in accessibility requirements so perhaps the current implementation is best?
